libr/util/sys.c

```diff
@@ -1143,7 +1143,21 @@ R_API bool r_sys_tts(const char *txt, bool bg) {
 R_API const char *r_sys_prefix(const char *pfx) {
 	static char prefix[1024] = {0};
 	if (!*prefix) {
+#if __WINDOWS__ && !CUTTER
+		char *path = r_sys_cmd_str ("where radare2", NULL, NULL);
```

Comment:
Does the `where` command works from r2 console?
Does it return `NULL` if r2 was not in `PATH` or an empty string?

Reply:
yes, its built in, if r2 is not in path, the return is an empty string

Reply:
I'm now using `r_sys_get_src_dir_w32()` that gets the path from the module